bench/BenchAll.hs

```diff
@@ -242,7 +247,7 @@ smallTraversalInput = S8.pack "The quick brown fox"
 main :: IO ()
 main = do
   mapM_ putStrLn sanityCheckInfo
-  Gauge.defaultMain
+  defaultMain
     [ bgroup "Data.ByteString.Builder"
```

Comment:
It would be good to move the non-`Builder`-related benchmarks out of this group, but this doesn't necessarily need to happen in this PR.

Reply:
What exactly looks non-`Builder`-related in this group?

Reply:
The `bgroup "intersperse"` and below seem unrelated.

Reply:
But they are not under `bgroup "Data.ByteString.Builder"`. 

```haskell
defaultMain
  [ bgroup "Data.ByteString.Builder" ...
  , bgroup "Data.ByteString.Builder.Prim" ...
  , bgroup "Data.ByteString.Builder.Prim.ASCII" ...
  , bgroup "intersperse" ... 
  , bgroup "partition" ... 
  , bgroup "sort" $ map (\s -> bench (S8.unpack s) $ nf S.sort s) sortInputs
  , bgroup "words" ...
  , bgroup "folds" ...
  , bgroup "findIndexOrLength" ...
  , bgroup "findIndex_" ...
  , bgroup "findIndexEnd" ...
  , bgroup "traversals" ...
  , benchBoundsCheckFusion
  , benchCSV
  , benchIndices
  ]
```

Reply:
Oops! Sorry for the confusion.